sssd_test_framework/hosts/samba.py

```diff
@@ -30,7 +30,7 @@ def __init__(self, *args, **kwargs) -> None:
 
         self._features: dict[str, bool] | None = None
 
-        self.admin: str = self.config.get("username", "Administrator")
+        self.adminuser: str = self.config.get("username", "Administrator")
```

Comment:
Do we know if `admin` is used anywhere?   Just adding a new var is easy but, will this change break something else?

Reply:
It is used in the framework in SambaGPO

Reply:
It is only used in SambaGPO in the framework, I did not found other locations